index.html

```diff
@@ -0,0 +1,120 @@
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+ <link href="https://fontawesome.com/icons/chevron-down?f=classic&s=regular">
+
+</head>
+<body>
+  <main>
+
+          <header >
+            <nav class="navbar">
+
+             <div class="hambuger">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+             </div>
+
+             <ul class="nav-menu">
+              <li class="nav-items"><a href="index.html">Home</a></li>
+              <li class="nav-items"><a href="about.html" >About Us</a></li>
+              <li class="nav-items"><a href="#" >Courses</a></li>
+              <li class="nav-items"><a href="#" >Contact</a></li>
+            </ul>
+            </nav>
+
+            <div class="header-1">
+              <nav class="header-1d">
+                <ul>
+                  <li><a href=""></a><i class="fa fa-facebook"></i></li>
+                  <li><a href=""></a><i class="fa fa-twitter"></i></li>
+                  <li><a href=""></a>English</li>
+                  <li><a href=""></a>My Page Layout</li>
+                  <li><a href=""></a>Logout</li>
+                </ul>
+              </nav>
+            </div>
+            <div class="header-2">
+              <nav class="desk-nav">
+                <div class="logo1"> <img src="./pics/partners/cck-logo1 1.png"></div>
+                <ul class="nav-d">
+                  <li class="item"><a href="about.html">About</a></li>
+                  <li class="item"><a href="index.html">Program</a></li>
+                  <li class="item"><a href="#">Join</a></li>
+                  <li class="item"><a href="#">Sponsor</a></li>
+                  <li class="item"><a href="#">Updates</a></li>
+                </ul>
+                <div class="logo2">Campaign</div>
+              </nav>
+            </div>
+          </header>
+
+          <section  class="home">
+              <div class="home-intro">
+                <p>"Hello! Kid Developers"</p>
+                <h1> INFORMATION TECHNOLOGY <br> FOR  KIDS </h1>
+              </div >
+              
+              <div class="intro-p">
+                <p> <u>Our vision</u>  : <strong> a source of empowerment for  digital education for children</strong>  accross Africa.
+                  We are open for the vulgarisation of digital education and arts creation for all our kids. As from:<strong> July 2023</strong>
+                </p>
+              </div>
+        
+            <div class="intro-dat">
+              <h2> 2023.03.07 (MON) ~ 07 (FRI)  </h2>
+              <p>&#64; The Afican kid's-Tech Center, Art Centre and more.</p>
+            </div>
```

Comment:
[Optional] Well done on applying the proper colors! However, I noticed that the elements in your section are not aligned correctly according to the design. In the design, all the elements should be left-aligned and placed horizontally form the same position. To achieve this, I recommend adjusting the layout of the elements in your code. Make sure they are aligned properly and placed in a horizontal line. This will help you achieve a consistent and visually appealing design. Keep up the great work! 👍🎨

| Your | Design|
| --- | --- |
| ![image](https://github.com/hydaline-code/Capstone_project_1/assets/55840999/2c266209-689d-4ced-86e7-d62cd21858e6) | ![image](https://github.com/hydaline-code/Capstone_project_1/assets/55840999/5f83474d-919f-4a37-af98-e5b9f7d792d5) |